tests/baselines/reference/booleanLiteralTypes1.types

```diff
@@ -1,32 +1,32 @@
 === tests/cases/conformance/types/literal/booleanLiteralTypes1.ts ===
 type A1 = true | false;
->A1 : boolean
+>A1 : A1
 >true : true
 >false : false
 
 type A2 = false | true;
->A2 : boolean
+>A2 : A2
 >false : false
 >true : true
 
 function f1() {
 >f1 : () => void
 
     var a: A1;
->a : boolean
->A1 : boolean
+>a : A1
+>A1 : A1
 
     var a: A2;
->a : boolean
->A2 : boolean
+>a : A1
+>A2 : A2
 
     var a: true | false;
->a : boolean
+>a : A1
 >true : true
 >false : false
 
     var a: false | true;
->a : boolean
+>a : A1

```

Comment:
This change looked wrong until I realized that this is the *fourth* declaration of `a`, and the first declaration has type `A1`, which is the type shown here.